package.json

```diff
@@ -50,6 +50,8 @@
     "webpack": "^5.0.0"
   },
   "dependencies": {
-    "ts-morph": "^14.0.0"
+    "@types/uuid": "^8.3.4",
+    "ts-morph": "^14.0.0",
+    "uuid": "^8.3.2"
```

Comment:
Can we instead just increase a counter? Like

```
window["${STORYBOOK_ANGULAR_ARG_TYPES}"]["${cmpName}${++GLOBAL_TYPE_COUNT}"]
```

It would remove a dependency in apps. If, for whatever reason, this does not work then lets use `uuid` 🙂

Reply:
That is a good idea. I do not find a reason why it shouldn't work if webpack performs its actions in a single thread. 
If webpack introduces some sort of parellelism or multi-processing we would need to stay with the uuid approach.

Reply:
I changed this to a counter, and seems to work fine. One problem that remains, disregarding if a counter or uuid is used:
At the moment, we just insert our code blocks (via the code-doc-dependency) at the end file. On reload, further code blocks are appended to the files, which are therefore slowly cluttered with more and more definitions. 
I do not know if this is a real problem (a few lines of code added each time should not be too bad) but if there is a simple way to avoid this i would do it anyways.

I thought about the problem for 15-20 minutes but could not come up with a super simple way to do it, since we are operating on a Webpack-Source, which is basically a string representation of the file (no AST or anything available). Best thing i found: Search with regex for approriate matches (including class names) and remove the code blocks.